lib/pbench/server/api/resources/query_apis/month_indices.py

```diff
@@ -15,12 +13,10 @@ class MonthIndices(ElasticBase):
 
     def __init__(self, config: PbenchServerConfig, logger: Logger):
         super().__init__(
-            config, logger, {},
+            config, logger, Schema(),
         )
```

Comment:
Can we do without the trailing comma?  (Let's save it for a code comment somewhere....)

Reply:
black put it there; not sure if it'll let me remove it. Maybe I can collapse this to one line?

Reply:
Collapsing it to one line is my goal.  :-)  So, try that (including removing the comma) and see if Black will leave it alone.